### DISCUSSION TITLE
A 68k assembler

### DISCUSSION THREAD

**Discussion by Pokeman2003** [vhelin/wla-dx#502] [Ideas]
Pretty simple idea. The Sega Genesis uses both a Motorola 68000 and a Zilog 80 for its processor, and since we already support the SNES(I'm aware it uses a 6502 normally, but we do have a SuperFX assembler here too), it would make sense to also support the Genesis with a 68000 assembler.

**Comment by vhelin** [comment]
Yeah, well, we have already this: https://github.com/vhelin/wla-dx/issues/170 ... Perhaps I'll work on it after adding support for function style defines (https://github.com/vhelin/wla-dx/issues/7) ?

**Reply by Pokeman2003** [reply]
That's on me. I looked before making this thread, but didn't look hard enough. I'm terribly sorry, sir.
